packages/binding.core/spec/styleBehaviors.ts

```diff
@@ -28,70 +29,76 @@ describe('Binding: CSS style', function () {
   })
 
   it('Should be able to use standard CSS style name (rather than JavaScript name)', function () {
-    var myObservable = new observable('red')
+    var myObservable = observable('red')
```

Comment:
also `let`.  I'll just note that generally and we can review/revisit in the future, it's not blocking, just noting.